Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -137,6 +137,260 @@ Once the permission `START_MAIN_ACTIVITY` has been created, apps can request it
 
 We recommend using a reverse-domain annotation when registering a permission, as in the example above (e.g. `com.domain.application.permission`) in order to avoid collisions with other applications.
 
+## Injection Flaws
+
+[Injection Flaws](https://owasp.org/www-community/Injection_Flaws "Injection Flaws") allow attackers to leverage the interconnection of applications and pass malicious code between the affected application and either another system (such as a server) or another connected client.
+
```

Comment:
Let's remove this one since we already have a more complete version in 0x04h-Testing-Code-Quality.md. I'll suggest a proper link below

```suggestion
```